index.js

```diff
@@ -17,7 +17,11 @@ module.exports = (css, settings) => {
 
   processedCss = result.stdout
 
-  if (processedCss instanceof Error || processedCss.name === 'CssSyntaxError') {
+  if (result.stderr) {
+    throw new Error(`postcss failed with ${result.stderr}`)
+  }
+
+  if (processedCss instanceof Error || processedCss.name === 'CssSyntaxError' ) {
```

Comment:
Now that we use a child process this is never going to be the case and should be moved to `processor.js`

Reply:
Good catch, fixed!